src/main/java/core/basesyntax/services/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.ReportWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class ReportWriterImpl implements ReportWriter {
```

Comment:
it can write not only reports
```suggestion
public class FileWriterImpl implements FileWriter {
```